src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form data-qa="big">
+      <div class="search__big">
+        <div class="search__big-icon">
+          <input
+            class="search__big-input"
+            type="text"
+            data-qa="keypress"
+            placeholder="Try “Los Angeles“"
+          />
```

Comment:
Consider adding a 'name' attribute to the input element for better form data handling. This is especially useful if the form data needs to be submitted or processed.